### DISCUSSION TITLE
在6.7日晚上更新后，好像卡主了，然后关闭后今天再开就白屏， 重装也是白屏

### DISCUSSION THREAD

**Discussion by qianye1904** [LmeSzinc/AzurLaneAutoScript#1204] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107124472/172627021-d096a2b4-fcff-4a8b-ace3-d4f27de2c5a8.png)
![image](https://user-images.githubusercontent.com/107124472/172627076-c69f311c-9c33-4378-9fa7-e3778fa7e8bb.png)
不知道这么上传图看不看得见。

**Comment by 18870** [comment, marked as answer]
#876
